### ISSUE TITLE
test/runner-win.c compile error on MinGW 

### ISSUE THREAD

**Issue by hnakamur** [joyent/libuv#583]
with error message.
test/runner-win.c:27:20: fatal error: crtdbg.h: No such file or directory

https://github.com/joyent/libuv/commit/8f1ec328656c8fb792fce4a346c11141ab2cd9f9

**Comment by bnoordhuis** [comment]
Does this patch fix it?

``` diff
diff --git a/test/runner-win.c b/test/runner-win.c
index 111a686..8ea459b 100644
--- a/test/runner-win.c
+++ b/test/runner-win.c
@@ -24,7 +24,10 @@
 #include <malloc.h>
 #include <stdio.h>
 #include <process.h>
-#include <crtdbg.h>
+
+#if !defined(__MINGW32__)
+# include <crtdbg.h>
+#endif

 #include "task.h"
 #include "runner.h"
```

**Comment by hnakamur** [comment]
No. I sent a pull request. Please see https://github.com/joyent/libuv/pull/585

**Comment by piscisaureus** [comment]
Fixed in 2684f876a.

**Comment by pH200** [comment]
Oops, sorry for breaking the build :p

Although libuv is only supported build with either MSVC or MinGW for Windows. Given the fact that `crtdbg.h` is an MSVC specific header,  maybe it would be more appropriate to write:
`#if defined(_MSC_VER)`
